### ISSUE TITLE
V1.4 does not calculate TROffset properlly at 59.9

### ISSUE THREAD

**Issue by JeanMTX** [ebu/pi-list#35]
Hi,
I have installed the container for version V1.4
When I load a PCAP for a 2110-20 stream for a video at 59.9 Hz the TROffset that is calculated by the application is wrong. This also results in VRX buffer analysis that is also wrong.
When I load a PCAP for a 2110-20 stream for a video at 50 Hz the TROffset that is calculated is correct.
> please note that in V1.3 both the 50 and 59 files are tagged as compliant to the VRX.

I am using a VB-440 to measure my TROffset and the values in both 50 and 59 are consistent and valid.
This screenshot shows the 59.9 analysis results. Notice the TROffset and VRX analisys. On the VB440 screen you will notice the path delay is listed at 577uSec. This is roughly equivalent to my TROffset.

![listntsc](https://user-images.githubusercontent.com/24941363/53588369-cb92c600-3b5a-11e9-8a9d-93218306fb8e.jpg)

In the below screenshot we see the results for 50Hz. TROffset is measured at 685 in LIST 1.4 and is listed as 688uSec on the VB440.
![listpal](https://user-images.githubusercontent.com/24941363/53588626-61c6ec00-3b5b-11e9-9213-9c7e46f48ef8.jpg)

I have attached a PCAP for the 2110-20 720p59 stream used in this test.

[720p59PCAP.zip](https://github.com/ebu/pi-list/files/2916351/82_162.zip)

**Comment by pedro-alves-ferreira** [comment]
Hi Jean,

Thanks for reporting this. I will check it a.s.a.p.

Kind regards,

Pedro

No dia 28/02/2019, às 18:25, JeanMTX <notifications@github.com<mailto:notifications@github.com>> escreveu:


Hi,
I have installed the container for version V1.4
When I load a PCAP for a 2110-20 stream for a video at 59.9 Hz the TROffset that is calculated by the application is wrong. This also results in VRX buffer analysis that is also wrong.
When I load a PCAP for a 2110-20 stream for a video at 50 Hz the TROffset that is calculated is correct.

please note that in V1.3 both the 50 and 59 files are tagged as compliant to the VRX.

I am using a VB-440 to measure my TROffset and the values in both 50 and 59 are consistent and valid.
This screenshot shows the 59.9 analysis results. Notice the TROffset and VRX analisys. On the VB440 screen you will notice the path delay is listed at 577uSec. This is roughly equivalent to my TROffset.

[listntsc]<https://user-images.githubusercontent.com/24941363/53588369-cb92c600-3b5a-11e9-8a9d-93218306fb8e.jpg>

In the below screenshot we see the results for 50Hz. TROffset is measured at 685 in LIST 1.4 and is listed as 688uSec on the VB440.
[listpal]<https://user-images.githubusercontent.com/24941363/53588626-61c6ec00-3b5b-11e9-9213-9c7e46f48ef8.jpg>

I have attached a PCAP for the 2110-20 720p59 stream used in this test.

720p59PCAP.zip<https://github.com/ebu/pi-list/files/2916351/82_162.zip>

**Comment by JeanMTX** [comment]
Thank you,

just to be clear. I have provided a 720p stream but this happens in all the resolutions I tried.

**Comment by pedro-alves-ferreira** [comment]
Hi Jean,

I think I found the issue. Are your pcap files timestamped with UTC or TAI?
I adjusted our pcap reader to add 37s to the timestamp of every packet and it now looks perfect:

![37s](https://user-images.githubusercontent.com/20090304/53605628-02df9200-3bb0-11e9-942a-e07bffd1030c.JPG)

Are you able to change the time reference? If not, we can add a toggle button to select UTC or TAI.

Kind regards,

Pedro
